app/src/models/BaseBox.ts

```diff
@@ -0,0 +1,16 @@
+import { Mesh, MeshBuilder, Scene } from "@babylonjs/core";
+
+class BaseBox extends Mesh {
+  public canBePicked: boolean;
+
+  constructor(name: string, scene: Scene) {
+    super(name, scene);
+    this.canBePicked = true;
+  }
+
+  public static create(name: string, size: number, scene: Scene): Mesh {
```

Comment:
I would put this code into constructor as well. No need to have separate create function when constructor is doing creation of objects. Also parameters are similar you can call this function code in constructor.